docs/Custom_Device_Tips_and_Tricks.md

```diff
@@ -0,0 +1,271 @@
+You must rebuild the **Configuration Release** and **Engine Release** build specifications to deploy the changes. You may then use the extra page as if it were generated by the niveristand-custom-device-wizard.
+
+The **[niveristand-custom-device-wizard](https://github.com/ni/niveristand-custom-device-wizard)** is open source. If you have any questions about what the wizard does, you can refer to the code as you would any other VI.
+
+### Custom Device XML
+
+The [Custom Device XML Tags](https://www.ni.com/documentation/en/veristand/latest/manual/custom-device-xml-tags/) define settings for a custom device. These elements, and [non-standard element types](https://www.ni.com/documentation/en/veristand/latest/manual/custom-device-xml-element-types/), are defined in the *Custom Device.xsd* schema located in the *[&lt;Common Data&gt;](https://www.ni.com/documentation/en/veristand/latest/manual/veristand-directories-aliases/)\Custom Devices* directory. You can open the file in an XML or text editor to read the schema and view the hierarchy.
+
+Consider the following example line from the *Custom Device.xsd* file:
+
+<xs:element minOccurs="0" name="ActionVIOnDelete" type="Path" />
+
+**A Line from the Custom Device XML Schema File**
+
+The name of this tag is *ActionVIOnDelete*. Adding this tag to the custom device XML runs a VI when the operator deletes the item from System Explorer. You may find experimenting with the custom device XML easier in an empty custom device.
+
+It may be helpful to explore VeriStand’s built-in components for examples on implementing XML features. The built-in components are found in &lt;Application Data&gt;\System Explorer\System Explorer Definition Files.
+
+If a tag is opened, use the format *</tag_name>* to close the tag. If a tag must be specified but has no value, you may use the format *<tag_name />* to open and close the tag at the same time. This format has the same effect as *<tag_name>tag value</tag_name>*.
+
+### Delete Protection
+
+You can add &lt;DeleteProtection&gt;true&lt;/DeleteProtection&gt; to any section in the [custom device XML](https://www.ni.com/documentation/en/veristand/latest/manual/custom-device-xml-tags/) to disallow deleting the item from the configuration tree in System Explorer.
+
+### Limiting Occurrences of the Custom Device
+
+If it doesn’t make sense to have more than **N** instances of the custom device in a single System Definition, you can limit the number of instances by adding **&lt;MaxOccurrence&gt;N&lt;/MaxOccurrence&gt;** to the [custom device XML](https://www.ni.com/documentation/en/veristand/latest/manual/custom-device-xml-tags/) underneath the device type.
+
+### Rename Protection
+
+There may be cases when you depend on a custom device item to have a certain name, and you’d like to prevent the operator from renaming the item. Add **&lt;DisallowRenaming*gt;true&lt;/DisallowRenaming&gt;** below the &lt;/Name&gt; tag for any page to prevent the operator from renaming the item.
```

Comment:
typo, on my side
```suggestion
There may be cases when you depend on a custom device item to have a certain name, and you’d like to prevent the operator from renaming the item. Add **&lt;DisallowRenaming&gt;true&lt;/DisallowRenaming&gt;** below the &lt;/Name&gt; tag for any page to prevent the operator from renaming the item.
```